### PR TITLE
DG for UI component

### LINKED ISSUE #160: UI UML + desc DG

### PR DESCRIPTION
**Pull request by Wincenttjoi**
- Changes link of UI components from addressbook to our group's
- Change diagram image .puml file

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -63,11 +63,11 @@ The sections below give more details of each component.
 ![Structure of the UI Component](images/UiClassDiagram.png)
 
 **API** :
-[`Ui.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/ui/Ui.java)
+[`Ui.java`](https://github.com/AY1920S2-CS2103T-F10-1/main/blob/master/src/main/java/seedu/address/ui/Ui.java)
```
Comment (bobbylaw) [inline docs/DeveloperGuide.md]: i think its the wrong one

Comment (Wincenttjoi) [reply docs/DeveloperGuide.md]: @zeranium97 Thanks! Updated

**Comment by zeranium97** [comment]
LGTM 👍

**Review by bobbylaw — APPROVED** [review]
LGTM 👍

**Pull request merged**